modules/magento/tests/playwright/MagentoErrors.spec.js

```diff
@@ -0,0 +1,167 @@
+import { expect, test } from '@playwright/test'
+import { makeProject, wrapConsole } from '@storefront-x/testing'
+
+test('magento error handling is working', async ({ page }) => {
+  await makeProject(
+    {
+      modules: [
+        '@storefront-x/base',
+        '@storefront-x/vue',
+        '@storefront-x/vue-router-simple',
+        '@storefront-x/vue-i18n',
+        '@storefront-x/vue-pinia',
+        '@storefront-x/base-commerce',
+        '@storefront-x/graphql',
+        '@storefront-x/magento',
+        '@storefront-x/cart',
+        '@storefront-x/checkout',
+        '@storefront-x/customer',
+        '@storefront-x/customer-magento',
+        [
+          'my-module',
+          {
+            components: {
+              'Error.vue': `
+                <template></template>
+                <script setup>
+                  import MagentoError from '#ioc/errors/MagentoError'
+
+                  throw new MagentoError({
+                    message: 'Error message has been sent!'
+                  })
+                </script>
+              `,
+            },
+            pages: {
+              'index.vue': `
+                <template>
+                  <Error />
+                  <p v-if="notification.notifications">{{ notification.notifications[0].message }}</p>
+                </template>
+                <script setup>
+                  import Error from '#ioc/components/Error'
+                  import useNotificationStore from '#ioc/stores/useNotificationStore'
+
+                  const notification = useNotificationStore()
+                </script>
+              `,
+            },
+            config: {
+              'MAGENTO_URL.ts': `export default 'https://be-sfx.demo.magexo.cloud'`,
```

Comment:
export default 'something non-sensical because we are actually not doing any network requests'